{
    "pexception-master/setup.py": {},
    "pexception-master/pexception/pexception.py": {
        "_trace": {
            "name": "_trace",
            "location": 13,
            "return": [
                "None"
            ],
            "arguments": {
                "exctype": [
                    "Type[BaseException]"
                ],
                "value": [
                    "BaseException"
                ],
                "tb": [
                    "types.TracebackType"
                ]
            }
        },
        "_parse": {
            "name": "_parse",
            "location": 27,
            "return": [
                "Tuple[(int, int, int, List[Tuple[(str, str, str, str)]])]"
            ],
            "arguments": {
                "stack": [
                    "List[str]"
                ]
            }
        },
        "_format": {
            "name": "_format",
            "location": 63,
            "return": [
                "List[str]"
            ],
            "arguments": {
                "max_file": [
                    "int"
                ],
                "max_func": [
                    "int"
                ],
                "max_line": [
                    "int"
                ],
                "frames": [
                    "List[Tuple[(str, str, str, str)]]"
                ]
            }
        },
        "_eprint": {
            "name": "_eprint",
            "location": 99,
            "return": [
                "None"
            ],
            "arguments": {
                "summaries": [
                    "List[str]"
                ]
            }
        },
        "hook": {
            "name": "hook",
            "location": 110,
            "return": [
                "None"
            ],
            "arguments": {
                "verbose": [
                    "bool"
                ],
                "align": [
                    "bool"
                ]
            }
        }
    },
    "pexception-master/pexception/__init__.py": {},
    "pexception-master/tests/fixtures.py": {},
    "pexception-master/tests/test_pexception.py": {
        "PExceptionTest.test_parse": {
            "name": "test_parse",
            "location": 11,
            "return": [
                "None"
            ],
            "arguments": {
                "self": []
            }
        },
        "PExceptionTest.test_format": {
            "name": "test_format",
            "location": 18,
            "return": [
                "None"
            ],
            "arguments": {
                "self": []
            }
        }
    }
}